tutorials/nextjs-app-serverless-functions-sqldb/index.mdx

```diff
@@ -563,10 +563,10 @@ To secure your deployment, we will now add a dedicated [IAM application](/iam/co
 
 - **Fine-tune deployment options** such as autoscaling, targeted regions, and more. You can find more information by typing `scw container deploy --help` in your terminal, or by referring to the [dedicated documentation](/serverless-containers/how-to/manage-a-container/).
 
-## Deploy a Next.js application using Terraform templates
+## Deploy a Next.js application using Terraform/OpenTofu templates
 
 <Message type="note">
-  You must have [Terraform](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli) installed and configured with [Scaleway Terraform Provider](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs) for this part.
+  You must have [Terraform/OpenTofu](https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli) installed and configured with [Scaleway Terraform/OpenTofu Provider](https://registry.terraform.io/providers/scaleway/scaleway/latest/docs) for this part.
```

Comment:
Have a distinction between the installation of terraform and the one from opentofu